discord/emoji.py

```diff
@@ -149,8 +150,11 @@ class Emoji:
         If this emoji is managed by a Twitch integration.
     guild_id: :class:`int`
         The guild ID the emoji belongs to.
+    created_by: :class:`User`
```

Comment:
I don't know why the attribute is named this way.

1) The API returns `user` not `created_by`
2) The UI uses `Uploaded By` not `Created By`

So where is this naming coming from?